db.go

```diff
@@ -15,20 +15,8 @@ import (
 
 const (
 	memtableSizeLimitBytes = 2 << 20 // 2MB
-	memtableFlushThreshold = 5 << 20
-	maxLevel               = 6
 )
 
```

Comment:
### Prerequisite Checks

- **Initialization of `db.levels`**: Ensure that `db.levels` is initialized before it is accessed. This is crucial to avoid runtime panics due to nil pointer dereferences. Consider adding a check at the beginning of functions that use `db.levels` to ensure it is not nil.

```go
if db.levels == nil {
    return false
}
```